catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/plugins/ParentMetacardStreamCreationPlugin.java

```diff
@@ -108,7 +108,7 @@ private void submitParentCreateRequest(Context context, CreateRequest createRequ
         catalogFramework.create(createRequest)
                 .getCreatedMetacards()
                 .forEach(createdMetacard -> {
-                    LOGGER.info("created parent metacard: metacard={}", createdMetacard.getId());
+                    LOGGER.debug("created parent metacard: metacard={}", createdMetacard.getId());
```

Comment:
This is just weird. I was going to suggest a similar solution to the one I proposed earlier, where the results were collected and logged once...but this call to `setParentMetacard`...it just keeps resetting to the last created card in the list. I'm not sure that's the intended behavior here. If it is, then this is a bit longer but more efficient:
```java
List<Metacard> createdMetacards = catalogFramework.create(createRequest)
        .getCreatedMetacards();
List<String> createdIds = createdMetacards.stream()
        .map(Metacard::getId)
        .collect(Collectors.toList());
LOGGER.debug("created parent metacards with ids: {}", createdIds);
context.setParentMetacard(createdMetacards.get(createdMetacards.size() - 1));
```